### DISCUSSION TITLE
Welcome to Printing-Pattern-Programs Discussions!

### DISCUSSION THREAD

**Discussion by aryashah2k** [aryashah2k/Programming-Patterns-And-Designs#67] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by neelima-girlcoder** [comment]
Hey @aryashah2k , do we work on the issues under this repo or start contributions directly to organization repos?

**Reply by aryashah2k** [reply]
I am planning to archive this repo since all of its code + the new updates are in process of migration from here to the individual repositories in the organization. I hope you are a part of that organization and you can directly work and create fresh issues on the org repos rather than here. I shall soon close all the issues here and redirect visitors to my organization PatternHouse
